k8s/demo/common/ccd/ccd-full.yaml

```diff
@@ -6,7 +6,19 @@ metadata:
   namespace: ccd
   annotations:
     flux.weave.works/automated: 'true'
-    flux.weave.works/tag.java: 'glob:prod-*'
+    flux.weave.works/tag.ccd-full-admin-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-api-gateway: glob:prod-*
+    flux.weave.works/tag.ccd-full-case-management-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-data-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-definition-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-dm-store: glob:prod-*
+    flux.weave.works/tag.ccd-full-draft-store: glob:prod-*
+    flux.weave.works/tag.ccd-full-em-annotation: glob:prod-*
+    flux.weave.works/tag.ccd-full-s2s: glob:prod-*
+    flux.weave.works/tag.ccd-full-sendletter: glob:prod-*
+    flux.weave.works/tag.ccd-full-user-profile-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-definition-importer: glob:prod-*
```

Comment:
Do we’ve to specify image param explicitly under each pod / container config in the flux file ?

Reply:
whats sendletter?

Reply:
nathan , that was mistake , fixed now